config/locales/user_mailer/en.yml

```diff
@@ -1,34 +1,34 @@
+---
 en:
   user_mailer:
     contact_request:
-      comments_header: "Comments:"
       comments: No comments left
-      email_or_phone: "Email address or phone number:"
+      comments_header: 'Comments:'
+      email_or_phone: 'Email address or phone number:'
       'no': no
```

Comment:
Same as the key, the value here (and below for `yes`) should be quoted, otherwise `t('user_mailer.contact_request.no')` will return the boolean value `false`.